README.md

```diff
@@ -75,13 +75,14 @@ let zfsDatasetsCommand:Command = Command(bash:"zfs list -t dataset")
 let zfsProcessInterface = ProcessInterface(command:zfsDatasetsCommand, stdout:.active(.lf), stderr:.active(.unparsedRaw))
 
 //launch the process. if you are running many concurrent processes (using most of the available resources), this is where your process will be queued until there are enough resources to support the launched process.
-let outputStreams = try await zfsProcessInterface.launch()
+try await zfsProcessInterface.launch()
 
 //handle lines of stdout as they come in
 var datasetLines = [Data]()
 for await outputLine in await zfsProcessInterface.stdout {
-	print("dataset found: \( String(data:outputLine, encoding:.utf8) )")
-	datasetLines += outputLine
+	guard let outputLineString = String(data: outputLine, encoding: .utf8) else { continue }
+	print("dataset found: \(outputLineString)")
+	datasetLines.append(outputLine)
```

Comment:
Using the `+=` operator here doesn't work. I get an error saying

<img width="971" alt="CleanShot 2023-07-27 at 14 15 10@2x" src="https://github.com/tannerdsilva/SwiftSlash/assets/923876/ef422ccc-e117-4102-8869-37eb63daa66f">

> Operator function '+=' requires the types 'Data' and 'Data.Element' (aka 'UInt8') be equivalent

Switching to using `.append()` fixes that.